File: est-modules/est-kotlin-support/est-typescript-sdk/examples/basic-usage.ts
import { EstClient, PluginMarketplaceClient } from '../src';

console.log('EST Framework TypeScript SDK - 基础使用示例');
console.log('='.repeat(60));

const client = new EstClient({ baseUrl: 'http://localhost:8080' });
console.log(`✓ EST客户端已创建: ${(client as any).config.baseUrl}`);

const marketplace = new PluginMarketplaceClient(client);
console.log('✓ 插件市场客户端已创建');

console.log('\n' + '='.repeat(60));
console.log('SDK功能:');
console.log('- 插件搜索');
console.log('- 插件详情获取');
console.log('- 插件版本管理');
console.log('- 评论功能');
console.log('- 插件安装/卸载');
console.log('- 插件更新');
console.log('\n提示: 请确保EST Framework服务已在 http://localhost:8080 启动');

File: est-modules/est-kotlin-support/est-python-sdk/examples/basic_usage.py
#!/usr/bin/env python3
"""
EST Framework Python SDK - 基础使用示例
"""

from est_sdk import EstClient, PluginMarketplaceClient
import sys


def main():
    print("EST Framework Python SDK - 基础使用示例")
    print("=" * 60)
    
    client = EstClient(base_url="http://localhost:8080")
    print(f"✓ EST客户端已创建: {client.base_url}")
    
    marketplace = PluginMarketplaceClient(client)
    print("✓ 插件市场客户端已创建")
    
    print("\n" + "=" * 60)
    print("SDK功能:")
    print("- 插件搜索")
    print("- 插件详情获取")
    print("- 插件版本管理")
    print("- 评论功能")
    print("- 插件发布")
    print("- 插件下载")
    print("\n提示: 请确保EST Framework服务已在 http://localhost:8080 启动")


if __name__ == "__main__":
    main()
